testsuite/cluster-test/src/experiments/load_test.rs

```diff
@@ -74,15 +85,30 @@ impl Experiment for LoadTest {
 
     async fn run(&mut self, context: &mut Context<'_>) -> anyhow::Result<()> {
         // spin up StubbedNode
-        let vfn = context.cluster.random_fullnode_instance();
-        let vfn_endpoint = format!("http://{}:{}/v1", vfn.ip(), vfn.ac_port());
-
-        let mut stubbed_node = StubbedNode::launch(vfn_endpoint).await;
-
+        let all_endpoints: Vec<_> = self
```

Comment:
we're trying to stress load one single node (N stubbed nodes -> 1 FN), this makes it a N 1:1 mapping which doesn't help.

Reply:
I c, my misunderstood, will correct this part